backend/src/test/java/ch/puzzle/okr/controller/QuarterControllerIT.java

```diff
@@ -39,31 +42,51 @@ class QuarterControllerIT {
             .withStartDate(LocalDate.of(2022, 9, 1))
             .withEndDate(LocalDate.of(2022, 12, 31))
             .build();
+
+    static QuarterDto quarter1Dto = new QuarterDto(quarter1
+            .getId(), quarter1.getLabel(), quarter1.getStartDate(), quarter1.getEndDate(), quarter1.isBacklogQuarter());
+
     static Quarter quarter2 = Quarter.Builder
             .builder()
             .withId(2L)
             .withLabel("GJ 22/23-Q3")
             .withStartDate(LocalDate.of(2023, 1, 1))
             .withEndDate(LocalDate.of(2023, 3, 31))
             .build();
+
+    static QuarterDto quarter2Dto = new QuarterDto(quarter2
+            .getId(), quarter2.getLabel(), quarter2.getStartDate(), quarter2.getEndDate(), quarter2.isBacklogQuarter());
+
     static Quarter backlogQuarter = Quarter.Builder
             .builder()
             .withId(BACK_LOG_QUARTER_ID)
-            .withLabel(BACK_LOG_QUARTER_LABEL)
+            .withLabel(BACKLOG_QUARTER_LABEL)
             .withStartDate(null)
             .withEndDate(null)
             .build();
-    static List<Quarter> quaterList = Arrays.asList(quarter1, quarter2, backlogQuarter);
+
+    static QuarterDto backlogQuarterDto = new QuarterDto(backlogQuarter.getId(),
+                                                         backlogQuarter.getLabel(),
+                                                         backlogQuarter.getStartDate(),
+                                                         backlogQuarter.getEndDate(),
+                                                         backlogQuarter.isBacklogQuarter());
+
+    static List<Quarter> quarterList = Arrays.asList(quarter1, quarter2, backlogQuarter);
+
+    static List<QuarterDto> quarterDtoList = Arrays.asList(quarter1Dto, quarter2Dto, backlogQuarterDto);
 
     @Autowired
     private MockMvc mvc;
     @MockitoBean
     private QuarterBusinessService quarterBusinessService;
+    @MockitoBean
+    private QuarterMapper quarterMapper;
 
     @DisplayName("Should get all quarters")
     @Test
     void shouldGetAllQuarters() throws Exception {
-        BDDMockito.given(quarterBusinessService.getQuarters()).willReturn(quaterList);
+        BDDMockito.given(quarterBusinessService.getQuarters()).willReturn(quarterList);
+        BDDMockito.given(quarterMapper.toDtos(any())).willReturn(quarterDtoList);
```

Comment:
```suggestion
        BDDMockito.given(quarterMapper.toDtos(quarterList)).willReturn(quarterDtoList);
```
Could be a bit more specific.